lib/game.rb

```diff
@@ -37,40 +37,36 @@ def start_game
 
   def user_input_cruiser_cells
     board = Board.new
-    puts "Enter the cells in which you would like to place your cruiser one at a time:"
+
     cruiser = Ship.new("Cruiser", 3)
     user_cruiser_cell_1 = "C5"
     user_cruiser_cell_2 = "E9"
     user_cruiser_cell_3 = "Z3"
     user_cruiser_cells = [user_cruiser_cell_1, user_cruiser_cell_2, user_cruiser_cell_3]
-
+# require "pry"; binding.pry
     while board.valid_placement_consecutive?(cruiser, user_cruiser_cells) == false
-
+      puts "Enter the cells in which you would like to place your cruiser one at a time:"
     user_cruiser_cell_1 = gets.chomp
 # require "pry"; binding.pry
-    while board.valid_coordinate?(user_cruiser_cell_1) == false
-      puts "This is not a valid coordinate. Please input a coordinate with a row letter between A and B and a column number between 1 and 4:"
-      user_cruiser_cell_1 = gets.chomp
-    end
-    # require "pry"; binding.pry
-    user_cruiser_cell_2 = gets.chomp
-    while board.valid_coordinate?(user_cruiser_cell_2) == false
-      puts "This is not a valid coordinate. Please input a coordinate with a row letter between A and B and a column number between 1 and 4:"
+      while board.valid_coordinate?(user_cruiser_cell_1) == false
+        puts "This is not a valid coordinate. Please input a coordinate with a row letter between A and B and a column number between 1 and 4:"
+        user_cruiser_cell_1 = gets.chomp
+      end
+      # require "pry"; binding.pry
       user_cruiser_cell_2 = gets.chomp
-    end
+      while board.valid_coordinate?(user_cruiser_cell_2) == false
+        puts "This is not a valid coordinate. Please input a coordinate with a row letter between A and B and a column number between 1 and 4:"
+        user_cruiser_cell_2 = gets.chomp
+      end
```

Comment:
Did you split up the process one by one in order to address the consecutive placement error?